### PR TITLE
Regression fix

### PR DESCRIPTION
**Pull request by HamidShojanazeri**
## Description

Fixing the dist inference test as it needs multi-gpu machine. Changing the regression runner to g5 instance being used for benchmarks.

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

File: test/pytest/test_distributed_inference_handler.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.mark.skipif(
```
Comment (msaroufim) [inline test/pytest/test_distributed_inference_handler.py:28]: You can break apart these lines to make it easier to read

```
pytest.mark.skipIf(condition_a)
pytest.mark.skipIf(condition_b)
test_function()
```

I had to triple read the not condition and where the parenthesis was to make sure there wasn't some bug that would run this on CPU

Would also be great to make sure this test gets skipped in the CPU regression tests

**Review by msaroufim — CHANGES_REQUESTED** [review]
c comment

File: test/pytest/test_distributed_inference_handler.py (hunk 1 above)
Comment (HamidShojanazeri) [reply test/pytest/test_distributed_inference_handler.py:28]: Thanks Mark, sure, just testing now if solve the issue the would fix regression_gpu.yaml back to run nightlies.

File: .github/workflows/regression_tests_gpu.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-  schedule:
-    - cron:  '15 9 * * *'
+  # schedule:
+  #   - cron:  '15 9 * * *'
 
 jobs:
   regression-gpu:
     # creates workflows for CUDA 11.6 & CUDA 11.7 on ubuntu
-    runs-on: [self-hosted, regression-test-gpu]
+    runs-on: [self-hosted, benchmark-gpu-new]
```
Comment (lxning) [inline .github/workflows/regression_tests_gpu.yml:11]: let's do not change the runner at this moment. we can skip the large model regression if # gpu < 4

**Pull request closed without merging**